rocAL/rocAL/source/loaders/image/image_read_and_decode.cpp

```diff
@@ -185,9 +211,54 @@ ImageReadAndDecode::load(unsigned char* buff,
             actual_height[file_counter] = max_decoded_height;
             file_counter++;
         }
+        skip_decode = true;
```

Comment:
why it is again set to true here?

Reply:
This is for different mode and decode is required here